src/zeit/content/volume/volume.py

```diff
@@ -115,8 +120,20 @@ def title(self, key):
 
     def __getattr__(self, key):
         """Interfere with zope.formlib and retrieve content via getitem."""
-        return zeit.cms.interfaces.ICMSContent(self.get(key), None)
+        return self.get(key)
 
     def __setattr__(self, key, value):
         """Interfere with zope.formlib and store content via setitem."""
         self[key] = value
+
+
+@grok.adapter(zeit.cms.content.interfaces.ICommonMetadata)
+@grok.implementer(zeit.content.volume.interfaces.IVolume)
+def retrieve_volume_using_info_from_metadata(context):
+    if (context.year is None or context.volume is None or
+            context.product is None or context.product.location is None):
+        return None
+    uniqueId = context.product.location.format(
```

Comment:
Das ist noch nicht ganz das gelbe vom Ei. :) Wir brauchen noch einen Test in der Art:
```
    def test_content_online_product_has_no_IVolume(self):
        # *All* content that is added in vivi gets year and volume from
        # zeit.cms.settings.interfaces.IGlobalSettings, so we need to ensure
        # that "online content" has no IVolume, only print content.
```